src/frontend/js/reducers/users-reducer.js

```diff
@@ -0,0 +1,28 @@
+import Immutable from 'immutable';
```

Comment:
+1 for using immutables

also, I see everyone has different patterns for defining reducers. interesting